Python/pywarpx/_libwarpx.py

```diff
@@ -456,8 +462,11 @@ def getistep(self, level=0):
         Parameter
         ---------
 
-        level : the refinement level to reference
+        level : int
```

Comment:
My own preference is to keep the parameter documentation on the one line (with extra lines indented). For me, this is especially important when there are multiple parameters, like with `add_particles`. It is a lot easier for me to parse the text in the original format.

Reply:
I definitely agree with you that in the `.py` file the inline formatting is much easier to read. The reason I changed it is because the auto generated documentation is not so readable with that formatting. The way it was renders the documentation as:
![old](https://user-images.githubusercontent.com/40245517/184448512-946f07c5-e11e-42f3-a973-ec5de3940dbc.png)

while the reformated doc-string produces:
![new](https://user-images.githubusercontent.com/40245517/184448545-3f89d9d6-b6d0-4dab-a6c5-009c2471fce1.png)

If there is a way to get the documentation formatted appropriately while keeping the inline doc-strings in the `.py` file I'd be all for that.

Reply:
In the picmi scripts, for example fields.py, uses this format

`- grid: grid object to be used by the solver (grid object)`

giving a bulleted list on the web page. I see now that your updated format is what is used in simulations.py, so it is consistent with that.

Reply:
The bulleted list is good for readability as well but I noticed there is a downside with it as well when dealing with multi-line descriptions as with `unique_particles` below
```
        - w                : Particle weights. Default = 0. (array or scalars)
        - unique_particles : Whether the particles are unique or duplicated on several
                             processes. Default = ``True``. (bool)
        - kwargs           : Containing an entry for all the extra particle attribute arrays.
                             If an attribute is not given it will be set to 0. (dict)
```
which renders as 
![old](https://user-images.githubusercontent.com/40245517/184451392-c8342901-63db-48cc-9bca-d4d7d2a562e8.png)

To render properly the docstring must be:
```
        - w                : Particle weights. Default = 0. (array or scalars)
        - unique_particles : Whether the particles are unique or duplicated on several
          processes. Default = ``True``. (bool)
        - kwargs           : Containing an entry for all the extra particle attribute arrays.
          If an attribute is not given it will be set to 0. (dict)
```

I think sticking with the formatting used in `simulations.py` will be best, especially with blank lines added between arguments, which helps a lot with readability in the `.py` file.

Reply:
Just for posterity I found out this specific way of formatting docstrings is called "Numpydoc".
https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats

Reply:
Ok, sounds good